### DISCUSSION TITLE
How come cloud custodian's version doesn't use semantic versioning (major.minor.patch) ?

### DISCUSSION THREAD

**Discussion by nitrocode** [cloud-custodian/cloud-custodian#9445] [General]
Hi. I noticed the version `0.9.35.0` has an extra `.0`. Why is that?

**Comment by kapilt** [comment]
its kinda of a historical raison at this point, no really good reason to keep it that I can see.

**Reply by kapilt** [reply]
the actual package version is without the suffix `.0` its only the releases in GitHub that have them.


```
❯ custodian version
0.9.36
```
